ui/src/dashboards/actions/index.js

```diff
@@ -123,12 +128,26 @@ export const templateVariableSelected = (dashboardID, templateID, values) => ({
   },
 })
 
+export const editTemplateVariableValues = (
+  dashboardID,
+  templateID,
+  values
+) => ({
+  type: 'EDIT_TEMPLATE_VARIABLE_VALUES',
+  payload: {
+    dashboardID,
+    templateID,
+    values,
+  },
+})
+
 // Async Action Creators
 
 export const getDashboardsAsync = () => async dispatch => {
   try {
     const {data: {dashboards}} = await getDashboardsAJAX()
     dispatch(loadDashboards(dashboards))
+    return dashboards
```

Comment:
funky to see a dispatch and then a return in a thunk, but works fine and the alternative (purely redux-state-change-driven) is not necessarily better, and definitely more work.